tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.IDataView.cs

```diff
@@ -76,6 +76,10 @@ public void TestIDataView()
             Assert.Equal("Bool", preview.ColumnView[13].Column.Name);
             Assert.Equal(true, preview.ColumnView[13].Values[0]);
             Assert.Equal(false, preview.ColumnView[13].Values[1]);
+
+            Assert.Equal("ArrowString", preview.ColumnView[14].Column.Name);
+            Assert.Equal("foo".ToString(), preview.ColumnView[14].Values[0].ToString());
```

Comment:
I would do AsMemory, like above with the normal string.

Reply:
For some reason, that just fails. I don't understand why. The test explorer says this:

```
Message: 
    Assert.Equal() Failure
    Expected: foo
    Actual:   foo
```

Reply:
That's odd - did you try debugging it? Why would it work for the normal string?